### ISSUE TITLE
Cookies are sent only for direct requests, not for page resources

### ISSUE THREAD

**Issue by LaserUnicorns** [AngleSharp/AngleSharp#420]
Hello. I have simple config:

```
var config = new Configuration()
                .WithDefaultLoader(setup => setup.IsResourceLoadingEnabled = true)
                .WithCookies();
```

When I send new request, cookies are sent only with page request, not with requests for page's resources (images, css files). Am I missing some config settings?

**Comment by FlorianRappl** [comment]
Which version of AngleSharp?

**Comment by LaserUnicorns** [comment]
0.9.8.1

**Comment by FlorianRappl** [comment]
Sounds strange to me then - I will investigate this. Thanks for the report 👍 !

**Comment by FlorianRappl** [comment]
Yes, the resource loader actually _never_ transports the cookie. As an intermediate work-around you could provide your own resource loader that inherits from `ResourceLoader` and overrides `DownloadAsync`. In this method you would add the `cookie` header with the value obtained via a `GetCookie(request.Address)` call.

A fix to this will be provided with the next release.

**Comment by FlorianRappl** [comment]
Fixed, see PR #423.
